packages/slice-machine/src/domain/slice.ts

```diff
@@ -281,3 +286,23 @@ export function rename(slice: ComponentUI, newSliceName: string): ComponentUI {
     },
   };
 }
+
+export function deleteRepeatableZone(args: DeleteRepeatableZoneArgs) {
```

Comment:
_✍️ potential change_

I know that `domain/slice.ts` isn't unit tested at all, not like `domain/customType.ts` that is fully unit tested. But what do you feel about adding a test for `deleteRepeatableZone` function? For me `domain` folder is one that is perfect for unit tests. WDYT?

Reply:
Yep, good idea! I added `@prismicio/mock` like the other PR to focus the tests on what we want to test. Let me know if you are good with that. 🙂

https://github.com/prismicio/slice-machine/pull/1334/commits/ffd17386fa91073e832582c3883c6def4168fce8

Reply:
Super nice!